### ISSUE TITLE
Imported css file returns undefined

### ISSUE THREAD

**Issue by AlexanderMattoni** [webpack-contrib/css-loader#331]
I'm working with the static website generator plugin (https://github.com/markdalgleish/static-site-generator-webpack-plugin) and React. Normally I don't have any problem with this but I can't figure out what's happening this time.

I've got the following loaders : 

```
            { test: /\.css$/, loader: ExtractTextPlugin.extract("style-loader", "css?modules&importLoaders=1&localIdentName=[path]___[name]__[local]___[hash:base64:5]") },
            { test: /\.scss$/, loader: ExtractTextPlugin.extract("style-loader", "css?modules&importLoaders=1&localIdentName=[path]___[name]__[local]___[hash:base64:5]!sass") },
```

and of course the plugin: 

```
    plugins: [
        new ExtractTextPlugin("css/[name].css"),
```

in my webpack config. This generates the files correctly. However, when the static website plugin does its thing to generate the HTML:

```
import styles from "./styles.scss";

console.log(styles);
```

styles ends up being undefined, despite having generated the correct files.

Am I missing something here? Can provide more info if needed.

Thanks!

**Comment by AlexanderMattoni** [comment]
It looks like 
```
import * as styles from "styles.scss"
```
works as expected. Are there plans to support default ES6 module imports?

**Comment by michael-ciniawsky** [comment]
Will be fixed by #402 in v1.0.0
